### PR TITLE
docs: update to `from prefect import get_client`

### LINKED ISSUE #9670: Change to `from prefect.client.orchestration import get_client`
### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to find a similar request and didn't find it.
- [X] I searched the Prefect documentation for this feature.

### Prefect Version

2.x

### Describe the current behavior

Although `from prefect.client.orchestration import get_client` works, however, it is not entirely correct, the IDE can also catch the issue:

image

A better way is using `from prefect.client.orchestration import get_client`

image

### Describe the proposed behavior

Update all

```py
from prefect.clientimport get_client
```

to

```py
from prefect.client.orchestration import get_client
```

### Example Use

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Hongbo-Miao**
### Example

Right now the repo is mixed using a few ways for users

```py
from prefect.client.orchestration import get_client
from prefect.client import get_client
from prefect import get_client
```

Based on @madkinsz 's comment at https://github.com/PrefectHQ/prefect/issues/9670#issuecomment-1557426333

This pull request updates all user facing code to

```py
from prefect import get_client
```

Close #9670

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.

### CODE REVIEW

File: src/prefect/cli/flow_run.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 from prefect.cli._types import PrefectTyper
 from prefect.cli._utilities import exit_with_error, exit_with_success
 from prefect.cli.root import app
-from prefect.client.orchestration import get_client
+from prefect import get_client
```
Comment (madkinsz) [inline src/prefect/cli/flow_run.py:19]: For all internal uses, we should probably still use the fully-specified `prefect.client.orchestration`.

Comment (Hongbo-Miao) [reply src/prefect/cli/flow_run.py:19]: Ah make sense. I will just update the doc for users.

File: src/prefect/__init__.py [hunk 2]
```diff
@@ -43,7 +43,7 @@
 from prefect.utilities.annotations import unmapped, allow_failure
 from prefect.results import BaseResult
 from prefect.engine import pause_flow_run, resume_flow_run
-from prefect.client.orchestration import get_client, PrefectClient
+from prefect import get_client, PrefectClient
```
[the remark this answers is not in the document]
Comment (Hongbo-Miao) [reply src/prefect/__init__.py]: Has reverted this change.

**Review by chrisguidry — APPROVED** [review]
Thank you for your contribution, @Hongbo-Miao!

**Pull request merged**